wsmaster/che-core-api-installer/src/main/java/org/eclipse/che/api/installer/server/model/impl/InstallerImpl.java

```diff
@@ -104,9 +103,11 @@ public InstallerImpl(String id,
                          Map<String, String> properties,
                          String script,
                          Map<String, ? extends ServerConfig> servers) {
+        Version.validate(version);
```

Comment:
I'm not sure that it is OK to do validation here. Maybe you should do it by InstallerRegistry, or InstallerService like workspace api does with workspace validation? See WorkspaceValidator.